### PR TITLE
Update `@digitalbazaar/webkms-client@9.0` to latest

### PR DESCRIPTION
**Pull request by JSAssassin**

### CODE REVIEW

File: test/package.json [hunk 1]
```diff
@@ -11,32 +11,32 @@
...
-    "bedrock-validation": "^5.2.0",
+    "bedrock-validation": "^5.2.2",
     "bedrock-veres-one-context": "^11.0.0",
-    "bedrock-zcap-storage": "^3.3.0",
+    "bedrock-zcap-storage": "^5.0.0",
     "cross-env": "^7.0.3",
-    "http-signature-zcap-invoke": "^3.0.0",
     "jsonld-signatures": "^9.3.0",
     "nyc": "^15.1.0",
     "zcap-context": "^1.2.1"
```
Comment (dlongley) [inline test/package.json]: `zcap-context` is deprecated, use `@digitalbazaar/zcap-context`. I've also found that a number of packages were including dependencies that weren't actually used. If this isn't imported / required, then please try removing it and seeing if the tests still pass.

**Review by dlongley — APPROVED** [review]
Provided that tests are passing here and in other applications that use this library, +1. See comments. Thanks!

File: test/package.json (hunk 1 above)
Comment (JSAssassin) [reply test/package.json]: Thanks, `zcap-context` and some other packages weren't being used. I've removed them 83b52ac.

**Review by dmitrizagidulin — APPROVED** [review]
:+1:

**Pull request merged**